app/views/comments/_comment_footer.html.erb

```diff
@@ -1,28 +1,34 @@
 <footer class="comment__footer print-hidden">
   <button
-    class="crayons-btn crayons-btn--ghost crayons-btn--icon-left crayons-btn--s mr-1 reaction-like inline-flex reaction-button"
+    class="crayons-tooltip__activator crayons-btn crayons-btn--ghost crayons-btn--icon-left crayons-btn--s mr-1 reaction-like inline-flex reaction-button"
     id="button-for-comment-<%= comment.id %>"
     data-comment-id="<%= comment.id %>"
-    title="heart"
+    aria-label="heart"
```

Comment:
This aria-label should ideally be "Like" - would you mind adding it to the i18n keys?